docs/Installation Guide.md

```diff
@@ -236,6 +236,13 @@ In addition to this, you would also have to populate the database for django-cit
 
     python manage.py migrate
     python manage.py cities_light
+    
+if ImportError: No module named cities_light ,this means cities_light has not installed properly.
+For this perform the following steps:
+
+    pip uninstall django-cities-light
+    pip install django-cities-light==3.2.0
+    pip install -r requirements.txt
```

Comment:
@Himanshi-Khandelwal Could you reason why this is happening though? Is it possible that this issue is due to trying to install packages independently first and then through the requirements file? Because in the slack chats you are referring to, they were trying to install it independently.
I will try uninstalling my setup and do it again following the guide. Meanwhile, please remove the fixes tag as the concerned issue is quite different. You can open an issue for the same.